cypress/components/popover-container/popover-container.cy.tsx

```diff
@@ -380,19 +427,16 @@ context("Test for Popover Container component", () => {
       cy.get(`[role="checkbox"]`).eq(0).check();
       cy.get(`[role="checkbox"]`).eq(1).check();
       cy.get(`[role="checkbox"]`).eq(2).check();
-      getDataElementByValue("main-text")
-        .eq(1)
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
+      getDataElementByValue("main-text").eq(1).click().as("maintext");
+      cy.get("@maintext").checkAccessibility();
```

Comment:
**suggestion:** Usually its not recommended to chain off action functions like `click()` [as described by cypress](https://docs.cypress.io/guides/core-concepts/retry-ability#Only-queries-are-retried), but it should be sufficient to call `checkAccessibility` like so:

```suggestion
      getDataElementByValue("main-text").eq(1).click();
      cy.checkAccessibility();
```